ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
+
+=== Being the Point of Contact for general issues related to the Supplementary Style Guide repository
```

Comment:
Maybe point of contact should be lowercase?

Reply:
+1

Reply:
It should:) Fixed. Thank you:)